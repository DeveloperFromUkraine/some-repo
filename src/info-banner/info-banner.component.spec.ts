import { ComponentFixture, TestBed } from "@angular/core/testing";
import { InfoBannerComponent } from "index";
import { ComponentTest } from "../../test/test-bed/component";

describe('InfoBannerComponent', () => {
    let fixture: ComponentFixture<InfoBannerComponent>;

    beforeEach(async () => {
        ComponentTest.createTestBed([], [InfoBannerComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(InfoBannerComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});