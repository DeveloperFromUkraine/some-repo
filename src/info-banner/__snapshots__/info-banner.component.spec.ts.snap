// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Banner should match snapshot 1`] = `
<ign-info-banner>
  <p>
    
    
    
  
  </p>
</ign-info-banner>
`;
